app/assets/stylesheets/earthworks.css

```diff
@@ -3,7 +3,7 @@
 }
 
 /* Selects both #main-content and navbar containers */
-#main-container, .navbar-search > * {
+#main-container, .navbar-search > *, .stanford-header-container, .stanford-footer-container {
```

Comment:
instead of adding these, could we just use the IDs/classes of the things that contain the container? so:
- `.masthead .container`
- `#pre-footer .container`
- `#su-footer .container`

I think those are easier to read and tell what they refer to.